test/pkg/environment/common/expectations.go

```diff
@@ -507,6 +527,19 @@ func (env *Environment) ConsistentlyExpectNoDisruptions(nodeCount int, duration
 	}, duration).Should(Succeed())
 }
 
+func (env *Environment) ConsistentlyExpectTaintedNodeCount(comparator string, count int, duration string) []*v1.Node {
```

Comment:
One thing I was curious about when I was making my changes to this function is why is this duration a string and not a time.Duration?

Reply:
It's a good call. I'm fine going to `time.Duration` if you want